package-scripts/datadog-agent/prerm

```diff
@@ -28,8 +28,11 @@ else
     exit 1;
 fi
 
-# Delete all.pyc files
+# Delete all.pyc files in the agent's dir
 find /opt/datadog-agent/agent -name '*.py[co]' -type f -delete || echo 'Unable to delete .pyc files'
 
-exit 0
+# Delete all the .pyc files in the embedded dir that are part of the agent's package
+# (i.e. not from packages installed manually by the user)
+cat /opt/datadog-agent/embedded/py_compiled_files.txt | xargs rm -f
```

Comment:
We should probably set permissions to `400 dd-agent:dd-agent`.

Also wondering if the removal command happens to fail (ie. file not found), would the uninstall operation be reported as a failure? I think it would, but not sure at all. Maybe worth wrapping it in a `File.exist?('/opt/datadog-agent/embedded/py_compiled_files.txt')`?